spec/lib/synced/strategies/full_spec.rb

```diff
@@ -133,18 +133,44 @@
         end
       end
 
-      describe "runs inside transaction" do
-        let(:remote_objects) { [
-          remote_object(id: 1, name: "test"),
-          remote_object(id: 1, name: "invalid")
-        ] }
+      describe "transactions handling" do
+        around do |example|
+          LosRecordsSyncSetupHelper.with_multipage_sync_crashing_on_second_page do
+            example.run
+          end
+        end
 
-        it "and doesn't save anything if saving one record fails" do
-          expect {
+        describe "without provided :transaction_per_page" do
```

Comment:
I think so, but I see that in most cases there are no contexts in the gem, so just made it consistent.